tests/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  irods-server:
+    container_name: irods-server
+    image: wsinpg/ub-18.04-irods-4.2.11
+    restart: always
+    ports:
+      - "1247:1247"
+
+  irods-clients: # setup (symlinks etc.) required is documented in irods-client readme of containers repo
+    container_name: irods-clients
+    image: wsinpg/md-bullseye-irods-clients-4.2.11
+    volumes:
+      - $PWD/../..:$PWD/../..
+      - $PWD:$HOME/.irods/
```

Comment:
Looking at PSD's repos e.g. https://github.com/sanger/crawler the docker-compose.yml is at the root, which makes the volume specs much nicer, IMO (I too tried having everything bundled in `tests` and I don't think it has much of an advantage). I think it's also good to adopt conventions from the other teams where that fits.